docs/release/trg-2/trg-2-3.md

```diff
@@ -2,22 +2,28 @@
 title: TRG 2.03 - Repo structure
 ---
 
-| Author                | Status | Created     | Post-History                               |
-|-----------------------|--------|-------------|--------------------------------------------|
-| Catena-X System Team  |        | 10-Nov-2022 | Initial release                            |
-| Catena-X System Team  |        | 24-Nov-2022 | add FOSS/Eclipse related basic files       |
-| Catena-X System Team  | Active | 25-Nov-2022 | adjustments to DEPENDENCY, LICENSE, NOTICE |
+ | Status | Created     | Post-History                               |
+|--------|-------------|--------------------------------------------|
+ | Active | 25-Nov-2022 | adjustments to DEPENDENCY, LICENSE, NOTICE |
+ |        | 24-Nov-2022 | add FOSS/Eclipse related basic files       |
+|        | 10-Nov-2022 | Initial release                            |
+
+## Why
+
+We are part of the Eclipse Foundation which requires complaince to the Eclipse Foundation rules. This also includes open and collaborative Open Source working model.
+
+Therefore, we need to follow the Eclipse Foundation and Open Source standards.
 
 ## Description
 
 :::info
 
-This TRG applies only to Eclipse-Tractusx GitHub repositories, not to Catenax-NG GitHub repositories. For Catenax-NG
+This TRG applies only to Eclipse-Tractusx GitHub repositories. For Catenax-ng
```

Comment:
Unsure, if we should keep the catenax-ng reference at all. There should be a section in 'oss' describing how to prepare the initial tractus-x move that contains references to legal docs templates

Reply:
Agree, catenax shouldn't be mentioned in this documentation.